{
  "dataType": "CVE_RECORD",
  "containers": {
    "adp": [
      {
        "title": "CVE Program Container",
        "references": [
          {
            "url": "https://github.com/kyverno/kyverno/security/advisories/GHSA-3hfq-cx9j-923w",
            "name": "https://github.com/kyverno/kyverno/security/advisories/GHSA-3hfq-cx9j-923w",
            "tags": [
              "x_refsource_CONFIRM",
              "x_transferred"
            ]
          }
        ],
        "providerMetadata": {
          "orgId": "af854a3a-2127-422b-91ae-364da2661108",
          "shortName": "CVE",
          "dateUpdated": "2024-08-02T21:16:42.282Z"
        }
      },
      {
        "title": "CISA ADP Vulnrichment",
        "metrics": [
          {
            "other": {
              "type": "ssvc",
              "content": {
                "id": "CVE-2023-47630",
                "role": "CISA Coordinator",
                "options": [
                  {
                    "Exploitation": "none"
                  },
                  {
                    "Automatable": "no"
                  },
                  {
                    "Technical Impact": "total"
                  }
                ],
                "version": "2.0.3",
                "timestamp": "2024-11-27T16:13:56.239908Z"
              }
            }
          }
        ],
        "providerMetadata": {
          "orgId": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
          "shortName": "CISA-ADP",
          "dateUpdated": "2024-11-27T16:13:59.914Z"
        }
      }
    ],
    "cna": {
      "title": "Attacker can cause Kyverno user to unintentionally consume insecure image",
      "source": {
        "advisory": "GHSA-3hfq-cx9j-923w",
        "discovery": "UNKNOWN"
      },
      "metrics": [
        {
          "cvssV3_1": {
            "scope": "UNCHANGED",
            "version": "3.1",
            "baseScore": 7.1,
            "attackVector": "NETWORK",
            "baseSeverity": "HIGH",
            "vectorString": "CVSS:3.1/AV:N/AC:H/PR:L/UI:R/S:U/C:H/I:H/A:H",
            "integrityImpact": "HIGH",
            "userInteraction": "REQUIRED",
            "attackComplexity": "HIGH",
            "availabilityImpact": "HIGH",
            "privilegesRequired": "LOW",
            "confidentialityImpact": "HIGH"
          }
        }
      ],
      "affected": [
        {
          "vendor": "kyverno",
          "product": "kyverno",
          "versions": [
            {
              "status": "affected",
              "version": "< 1.10.5"
            }
          ]
        }
      ],
      "references": [
        {
          "url": "https://github.com/kyverno/kyverno/security/advisories/GHSA-3hfq-cx9j-923w",
          "name": "https://github.com/kyverno/kyverno/security/advisories/GHSA-3hfq-cx9j-923w",
          "tags": [
            "x_refsource_CONFIRM"
          ]
        }
      ],
      "descriptions": [
        {
          "lang": "en",
          "value": "Kyverno is a policy engine designed for Kubernetes. An issue was found in Kyverno that allowed an attacker to control the digest of images used by Kyverno users. The issue would require the attacker to compromise the registry that the Kyverno users fetch their images from. The attacker could then return an vulnerable image to the the user and leverage that to further escalate their position. As such, the attacker would need to know which images the Kyverno user consumes and know of one of multiple exploitable vulnerabilities in previous digests of the images. Alternatively, if the attacker has compromised the registry, they could craft a malicious image with a different digest with intentionally placed vulnerabilities and deliver the image to the user.  Users pulling their images by digests and from trusted registries are not impacted by this vulnerability. There is no evidence of this being exploited in the wild. The issue has been patched in 1.10.5. All users are advised to upgrade. There are no known workarounds for this vulnerability."
        }
      ],
      "problemTypes": [
        {
          "descriptions": [
            {
              "lang": "en",
              "type": "CWE",
              "cweId": "CWE-345",
              "description": "CWE-345: Insufficient Verification of Data Authenticity"
            }
          ]
        }
      ],
      "providerMetadata": {
        "orgId": "a0819718-46f1-4df5-94e2-005712e83aaa",
        "shortName": "GitHub_M",
        "dateUpdated": "2023-11-14T20:59:46.100Z"
      }
    }
  },
  "cveMetadata": {
    "cveId": "CVE-2023-47630",
    "state": "PUBLISHED",
    "dateUpdated": "2024-11-27T16:14:09.547Z",
    "dateReserved": "2023-11-07T16:57:49.244Z",
    "assignerOrgId": "a0819718-46f1-4df5-94e2-005712e83aaa",
    "datePublished": "2023-11-14T20:59:46.100Z",
    "assignerShortName": "GitHub_M"
  },
  "dataVersion": "5.1"
}